cache/application.go

```diff
@@ -9,7 +9,6 @@ import (
 type Application struct {
 	cache.Driver
 	config config.Config
-	driver Driver
```

Comment:
We can remove the Driver interface, then change `DriverImpl` to `Driver`, change here to `driver * Driver`, and keep `instance, err := app.driver.New(name)` in line 38.